_posts/2017-01-30-inc-cmd-lessonslearned.md

```diff
@@ -0,0 +1,79 @@
+
+- Assigning tasks to a specific person (by name) or function,
+- Stating clearly the objective,
+- Expecting the receiver to acknowledge and repeat back a summary of the assignment, and
+- Giving a specific time to complete the task.
+
+> IC: “Network ops, International tribe are seeing are seeing packet loss on their office VPN to the DC. Can you drop off the call, take a look and get back to me in at 0915?”
+
+> Netops: “Will do, Network ops on-call dropping off the call to link up International and investigate their VPN, will report back to you in 15 minutes.”
+
+#### Understand battle rhythm
+An IC is on a constant running battle between triaging the problem, tasking resources, communicating with customers and stakeholders and considering contingency plans. Regularly recap the situation for the people on the conference call or Slack channel to focus them, this can be done with a CAN report to keep things in a 'standard' format so everyone understands what is going on.
+
+-	Conditions – Current status
+-	Actions – What is being done or needs to be done?
+-	Needs – What resources are needed or other action items required?
+
+**Note:** Identify who you are talking to at the start of your sentence to get their attention, and use pauses wisely to allow people to digest what you are communicating
+
+For example:
+
+> **C:** “IC: Okay everyone listen in. Let’s take a look at where we are with this incident. Incident has been open for 30 minutes situation as I understand it is that we have narrowed down the problem to the DC firewall blocking access to the International office.”
+
+> **A:** “IC: Network ops have looked at it and are 90% certain that it is a problem with a runaway process on the firewall. They are currently working to fail over the cluster to resolve this within the next 10 minutes.”
+
+> **N:** “IC: I want to keep everyone on the conf call for now until we are certain of the fix. While we are waiting, lets discuss our next most likely root cause candidate and what our plan B will be.”
+
+#### On that subject, have a plan B!
+When your team have come up with plan A and are focussing on that you need to be thinking about plan B and C, getting resource lined up to support them and switching assets to support a secondary or tertiary plan when appropriate in a timely manner. Do not be afraid to adapt or change your plans where it is justified to do so. Your main focus should always be on restoring service as quickly as possible and it can sometimes be a tough call whether to ask a team to 'switch fire' to another plan. Where possible have multiple teams working on different plans rather than having a single team context switching to come up with multiple plans.
```

Comment:
"focusing" used before and in previous posts on this blog. Drop the second "S" from this instance.